README.md

```diff
@@ -1 +1,53 @@
-# babble
\ No newline at end of file
+# Babble
+![image of babble landing page]() <br />
+[Babble](https://babble-lilac.vercel.app/) is a video chat web app with Zoom integration and synchronous text workspace support.
+
+## Table of Contents
+* [Description](#description)
+* [Demo](#demo)
+* [Site](#site)
+* [Technologies](#technologies)
+* [Usage](#usage)
+* [Contributing](#contributing)
+* [Feature Requests and Bug Reports](#feature-requests-and-bug-reports)
+* [License](#license)
+
+## Description
+[Babble](https://babble-lilac.vercel.app/) provides live video chat and text field services. It enables people from around the world to work together. Users can create rooms with passwords, invite people to their rooms, live video chat and type in a shared text field.
+
+## Demo
+Live demo available here: https://babble-lilac.vercel.app/
+
+## Site
+
+### Landing Page
+![image of babble landing page]()
+
+### Rooms
+![image of babble rooms page]()
+
+## Technologies
+* Vue.js with Tailwind CSS
+* [Zoom SDK](https://marketplace.zoom.us/docs/sdk/native-sdks/introduction)
```

Comment:
Is it cool if this gets changed to be [PeerJS](https://peerjs.com/)? That's what I'm working on instead of the Zoom SDK at the moment!